src/Utils.jl

```diff
@@ -342,6 +342,31 @@ Compute the time step `dt` for the velocity field `S.V` and the diffusive maximu
     dt_adv = mapreduce(x -> x[1] * inv(maximum(abs.(x[2]))), max, zip(di, V)) * n
     return min(dt_diff, dt_adv)
 end
+"""
+    compute_dt(S::StokesArrays, di, igg)
+
+Compute the time step `dt` for the velocity field `S.V` for a regular gridwith grid spacing `di`.
+The implicit global grid variable `igg` implies that the time step is calculated globally and not
+separately on each block.   
+"""
+@inline compute_dt(S::StokesArrays, di, igg) = compute_dt(@velocity(S), di, Inf, igg)
```

Comment:
I think there may be a method ambiguity here, better add the type of `igg` in the signature call.